adafruit_esp32spi/adafruit_esp32spi.py

```diff
@@ -131,6 +131,97 @@
 # pylint: disable=too-many-lines
 
 
+class ESP_Network:
```

Comment:
Built this to match `wifi.radio` the best I could

Reply:
Maybe call this `Radio`?

Reply:
I tried to match CP WiFi, and there it's called `Network`: https://docs.circuitpython.org/en/latest/shared-bindings/wifi/index.html#wifi.Network

Reply:
@dhalbert do you think just `Network`?

Reply:
Yep, that's exactly what I was going to say. If it quacks like a `Duck`, it doesn't have to be an `ESP_Duck`.

Reply:
Fixed here: 2da98438cf81145166f89cd7fa3969e32ca186af